glide.lock

```diff
@@ -1860,7 +1860,7 @@ imports:
   - pkg/util/proto/testing
   - pkg/util/proto/validation
 - name: k8s.io/kubernetes
-  version: d8b46d72a413981e6ea99605dcef3b8ca3f2223f
+  version: 6112ef84c44b5db537e9fadffcafa06b70ef4e5b
```

Comment:
@soltysh as usual i have to ask why we have a k8s commit changing w/o k8s content changing...

Reply:
@bparees we're syncing our vendored k8s out to openshift/kubernetes repo, but only contents, so the contents is in sync between the two, hashes not quite.